src/main/java/uk/gov/digital/ho/hocs/domain/queue/migration/MigrationService.java

```diff
@@ -38,40 +44,121 @@ public class MigrationService {
     public MigrationService(MigrationCaseworkClient migrationCaseworkClient,
                             RequestData requestData,
                             ObjectMapper objectMapper,
-                            MessageLogService messageLogService) {
+                            MessageLogService messageLogService,
+                            DocumentClient documentClient) {
         this.migrationCaseworkClient = migrationCaseworkClient;
         this.requestData = requestData;
         this.objectMapper = objectMapper;
         this.messageLogService = messageLogService;
+        this.documentClient = documentClient;
     }
 
     public void createMigrationCase(MigrationData migrationCaseData, MigrationCaseTypeData migrationCaseTypeData) {
+        UUID caseId;
+        UUID stageId;
+        ResponseEntity<CreateMigrationCaseResponse> caseResponseEntity;
+
+        // case
         try {
             var migrationRequest = composeMigrateCaseRequest(migrationCaseData, migrationCaseTypeData);
-            CreateCaseworkCaseResponse caseResponse = migrationCaseworkClient.migrateCase(migrationRequest);
-            messageLogService.updateCaseUuidAndStatus(requestData.getCorrelationId(), caseResponse.getUuid(), Status.CASE_CREATED);
+            caseResponseEntity = migrationCaseworkClient.migrateCase(migrationRequest);
```

Comment:
Could this just end be of type CreateMigrationCaseResponse? Even better than calling `getBody` we could have the `CaseworkClient` do the unpacking?